.github/scripts/bash/build.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+source `dirname ${BASH_SOURCE[0]}`/../config.sh
+
+BASH_PATH=${BASH_PATH:-~/$TARGET/bash}
+BASH_BUILD_PATH=$BUILD_PATH/bash
+BASH_SOURCE_PATH=$SOURCE_PATH/bash
+
+PATH="$TOOLCHAIN_PATH/bin:$TOOLCHAIN_PATH/lib/gcc/$TARGET/:$TOOLCHAIN_PATH/lib/gcc/$TARGET/$GCC_VERSION/:$PATH"
+LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$TOOLCHAIN_PATH/bin
+
+mkdir -p $BASH_BUILD_PATH
+cd $BASH_BUILD_PATH
+
+if [[ "$RUN_CONFIG" = 1 ]] || [[ ! -f "$BASH_BUILD_PATH/Makefile" ]]; then
+    echo "::group::Configure Bash"
+        rm -rf $BASH_BUILD_PATH/*
+
+        CFLAGS="-std=gnu89 -DNEED_EXTERN_PC" \
```

Comment:
Is `-DNEED_EXTERN_PC` a workaround that should be removed in the future? If yes, I'd prefer having this workaround in Bash source code so all the workarounds are on one place.

Reply:
It looks like an applicable configuration for aarch64-w64-mingw32

Reply:
Please explain?

Reply:
For some targets it should use "extern" as the variables are already defined.
This should be investigated more later, especially if it breaks something.
https://github.com/Windows-on-ARM-Experiments/bash/blob/6794b5478f660256a1023712b5fc169196ed0a22/lib/readline/terminal.c#L105

Reply:
In both cases, when it will investigated be resolved, the change here is supposed to be removed as it will be added either to compiler or to Bash source. Cannot we just avoid the future unnecessary PR on the build scripts if we add that workaround to the Bash source today?

Reply:
Moved.